x/ccv/provider/keeper/proposal_test.go

```diff
@@ -496,7 +496,7 @@ func TestPendingConsumerRemovalPropDeletion(t *testing.T) {
 	ctx = ctx.WithBlockTime(time.Now().UTC())
 
 	propsToExecute := providerKeeper.ConsumerRemovalPropsToExecute(ctx)
-	// Delete consumer removal proposals, same as what would be done by IteratePendingConsumerRemovalProps
+	// Delete consumer removal proposals, same as what would be done by IterateMatureConsumerRemovalProps
```

Comment:
Outdated comment prob? #370 addresses this naming
```suggestion
	// Delete consumer removal proposals, same as what would be done by BeginBlockCCR
```